src/Cognito.re

```diff
@@ -83,4 +165,50 @@ let signUp =
         });
       };
     });
+};
+
+open SignInHelper;
+external makeSignInResponse: 't => signInResponseDecoder = "%identity";
+external toJsDict: 'a => Js.Dict.t(Js.Json.t) = "%identity";
```

Comment:
I'd prefer boilerplate dict building over bucklescript magic for something this simple, like: 

https://github.com/pckilgore/reason-cognito/blob/f1b437a8eaafd19abafae9443069537bbce03ec0/src/Cognito.re#L142-L146

Reply:
So this is exactly magic. The result of `deriving abstract` is a valid, testable js object. I did because I couldn't figure out how to get the AuthParameter object into the `params` dict. As I say that, I think I did it somewhere else in this same file so will refactor.

Reply:
Done.